src/earn/EarnEnterAmount.tsx

```diff
@@ -286,6 +286,16 @@ function EarnEnterAmount({ route }: Props) {
     prepareTransactionsResult.type === 'possible' &&
     prepareTransactionsResult.transactions.length > 0
 
+  const allRewardsPositions = useSelector(positionsWithBalanceSelector)
```

Comment:
these are not all rewards right?

Reply:
right, updated to `allPositionsWithBalance` in [68e6ade](https://github.com/valora-inc/wallet/pull/6169/commits/68e6ade37b66eba67cbcce22b6e25a8cf625dfa9)